src/uct/ib/mlx5/dv/ib_mlx5_ifc.h

```diff
@@ -92,7 +92,6 @@ enum {
 
 enum {
     UCT_IB_MLX5_CAP_GENERAL   = 0x0,
-    UCT_IB_MLX5_CAP_ODP       = 0x2,
```

Comment:
Maybe keep it, for future ODP?

Reply:
let's remove; ODP would be driver API, not FW